index.js

```diff
@@ -0,0 +1,53 @@
+const arr = JSON.parse(localStorage.getItem('formData')) || [];
+
+const form = document.querySelector('#form');
+const add = document.getElementById('addBtn');
+// const title = document.getElementById('title');
+// const author = document.getElementById('author');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+});
+
+add.addEventListener('click', () => {
+  const formData = {
+    title: document.getElementById('title').value,
+    author: document.getElementById('author').value,
+  };
+  arr.push(formData);
+
+  localStorage.setItem('formData', JSON.stringify(arr));
+});
```

Comment:
- [OPTIONAL] Since you are already listening for a submit event on the form, you can remove the `event listener on the add button` and just create an `add function` and then call the function inside the `form submit event`. :smile: :+1: 
- - Also it would give a good user experience if you cleared the form input after the user submits the form. :smile: :wave: